### PR TITLE
Update to 10.2.0

### PR DESCRIPTION
**Pull request by cvium**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
-# emby-auto-organize
-Auto-organize plugin for Emby Server
+<h1 align="center">Jellyfin AutoOrganize Plugin</h1>
```
Comment (hawken93) [inline README.md:1]: I don't really like html in md, because a principle of md is that it should read well without the rendering. Just an opinion

**Review by hawken93 — COMMENTED** [review]
thank you for this PR :)

File: Emby.AutoOrganize/Emby.AutoOrganize.csproj [hunk 2]
```diff
@@ -35,8 +35,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.1.0" />
-    <PackageReference Include="Jellyfin.Naming" Version="10.1.0" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.2.0" />
```
Comment (anthonylavado) [inline Emby.AutoOrganize/Emby.AutoOrganize.csproj]: `<PackageReference Include="Jellyfin.Controller" Version="10.2.0.1" />`

File: Emby.AutoOrganize/Emby.AutoOrganize.csproj [hunk 3]
```diff
@@ -35,8 +35,8 @@
...
+    <PackageReference Include="Jellyfin.Naming" Version="10.2.0" />
```
Comment (anthonylavado) [inline Emby.AutoOrganize/Emby.AutoOrganize.csproj]: `<PackageReference Include="Jellyfin.Naming" Version="10.2.0.1" />`

**Review by anthonylavado — CHANGES_REQUESTED** [review]
~ignore this one~

File: Emby.AutoOrganize/Emby.AutoOrganize.csproj (hunk 2 above)
Comment (anthonylavado) [inline Emby.AutoOrganize/Emby.AutoOrganize.csproj]:
```suggestion
    <PackageReference Include="Jellyfin.Controller" Version="10.2.0.1" />
```

File: Emby.AutoOrganize/Emby.AutoOrganize.csproj (hunk 3 above)
Comment (anthonylavado) [inline Emby.AutoOrganize/Emby.AutoOrganize.csproj]:
```suggestion
    <PackageReference Include="Jellyfin.Naming" Version="10.2.0.1" />
```

**Review by anthonylavado — CHANGES_REQUESTED** [review]
Just minor things due to the bad NuGet packages

File: README.md (hunk 1 above)
Comment (cvium) [reply README.md:1]: It's actually a copy of the README from the main repo

**Review by anthonylavado — APPROVED** [review]

**Review by joshuaboniface — APPROVED** [review]

**Pull request merged**